vpp-manager/vpp_runner.go

```diff
@@ -45,8 +45,9 @@ import (
 )
 
 var (
-	fakeNextHopIP4    = net.ParseIP("0.0.0.0")
-	fakeNextHopIP6    = net.ParseIP("::")
+	fakeNextHopIP4s   = map[string]net.IP{"eth0": net.ParseIP("0.0.0.0")}
```

Comment:
We should probably directly make this a property of the uplink interface instead of a dedicated map